nicegui/elements/div.py

```diff
@@ -0,0 +1,12 @@
+import justpy as jp
+
+from .element import Element
+from ..auto_context import ContextMixin
+
+
+class Div(Element, ContextMixin):
```

Comment:
I find `ui.div` to be not ideal as an element name for NiceGUI. It's too close to low-level html and not high-level UI language. Couldn't we not just use `ui.group`? Maybe instead of the MixIn we can simply allow instantiating the Group-Element we already have....